contract-sdk/types/src/message.rs

```diff
@@ -65,6 +71,13 @@ impl CallResult {
     }
 }
 
+/// Instantiate call result.
+#[derive(Clone, Debug, cbor::Encode, cbor::Decode)]
+pub struct InstantiateResult {
```

Comment:
So this is specific to the `contracts.Instantiate` module call. Maybe these should go into their own module instead of being part of `message`? It would make sense to have things namespaced by module name, e.g. `crate::modules::contracts::*`. And then in the documentation for the `modules` module explain that these are the types for easier calling into existing SDK modules.

Reply:
Done moved this type into `contract-sdk/modules/contracts.rs`